PHEx/src/background.js

```diff
@@ -1,18 +1,19 @@
 const browser = chrome || browser;
-// debug will switch to redirector on localhost instead of prodigyhacking
+
+// If debug is enabled, the extension will use localhost instead of the Prodigy Hacking Organization CDN.
 const debug = false;
 
 // Ignore X-Frame Headers
 chrome.webRequest.onHeadersReceived.addListener(
 	(details) => ({ responseHeaders: details.responseHeaders.filter((header) => !["content-security-policy", "x-frame-options",].includes(header.name.toLowerCase())) }),
 	{ urls: ["<all_urls>"] }, ["blocking", "responseHeaders"]
 );
+
 // Redirect Requests
 chrome.webRequest.onBeforeRequest.addListener(details => {
 	const redirectorDomain = debug ? "http://localhost:1337" : "https://prodigyhacking.ml"
 
 	if (details.url.startsWith("https://code.prodigygame.com/code/") && details.url.includes("/game.min.js")) {
-		// if anybody curses my name, you might as well curse at me on twitter lol, I'm @PatheticMustan
```

Comment:
sad

Reply:
i mean im fine with adding it back as long as you add a link to https://digitalteal.com too!